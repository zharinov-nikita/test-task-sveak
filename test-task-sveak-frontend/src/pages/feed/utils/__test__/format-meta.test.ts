import { test, describe, expect } from 'vitest'
import { utilFormatMeta } from '../format-meta.util'

describe('utilFormatMeta', () => {
    test(() => {
        const author = 'John Doe'
        const published = '2023-09-15'
        const rating = 4.5
        const formattedMeta = utilFormatMeta(author, published, rating)
        expect(formattedMeta).toMatchSnapshot()
    })
})
