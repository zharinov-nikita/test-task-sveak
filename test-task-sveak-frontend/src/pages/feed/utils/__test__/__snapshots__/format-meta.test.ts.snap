// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`utilFormatMeta > должна правильно форматировать метаданные с отсутствующими полями 1`] = `"Alice Smith •"`;

exports[`utilFormatMeta > должна правильно форматировать метаданные с пустыми полями 1`] = `""`;

exports[`utilFormatMeta > должна правильно форматировать метаданные с числовыми значениями 1`] = `"Bob Brown • 1631731200 • 3.75"`;

exports[`utilFormatMeta > должна правильно форматировать метаданные со всеми полями 1`] = `"John Doe • 2023-09-15 • 4.5"`;
